packages/jetpack-ai-calypso/src/logo-generator/components/prompt.scss

```diff
@@ -1,6 +1,64 @@
+.jetpack-ai-logo-generator__prompt {
+	display: flex;
+	flex-direction: column;
+	gap: 8px;
+	font-size: $font-body-small;
+}
+
 .jetpack-ai-logo-generator__prompt-header {
 	display: flex;
 	justify-content: space-between;
 	align-items: flex-start;
 	align-self: stretch;
+
+	.jetpack-ai-logo-generator__prompt-label {
+		font-weight: 500;
+	}
+
+	.jetpack-ai-logo-generator__prompt-actions {
+		display: flex;
+		font-size: $font-body-extra-small;
+		line-height: 20px;
+	}
+}
+
+.jetpack-ai-logo-generator__prompt-query {
+	display: flex;
+	padding: 8px 8px 8px var(--grid-unit-15, 16px);
+	justify-content: space-between;
+	align-items: flex-end;
+	align-self: stretch;
+	border-radius: calc(4px * 2);
+	border: 1px solid var(--Gutenberg-Gray-400, #ccc);
```

Comment:
The colors on Figma do not match the ones available on Calypso. This should probably be `--studio-gray-400`.